Getting-Started.ipynb

```diff
@@ -105,7 +102,8 @@
     "---\n",
     "Lets focus on *numaster*, the master catalog for *NuSTAR*, and search for data on some object, say the X-ray binary **Cyg X-1**.\n",
     "\n",
-    "We user `astropy` to resolve the name into positional coordinate."
+    "We use `astropy` to resolve the name into positional coordinate.\n",
+    "We specify the service we want to use as `conesearch`"
```

Comment:
Minor thing: I'm wondering if there should be a period at the end of this string.